workers.py

```diff
@@ -113,18 +117,23 @@ def worker() -> None:
             rows,
         )
 
-        data_frame = rad.inventory_data_to_pandas(batch_data, *FEATURE_LIST)
-        data_frame, _mapping = rad.preprocess(data_frame)
-        isolation_forest = rad.IsolationForest(
-            data_frame,
-            num_trees,
-            sample_size,
-        )
-        results = isolation_forest.predict(
-            data_frame,
-            min_score=env['min_score'],
-        )
-
+        with METRICS['preparation_time'].time():
+            data_frame = rad.inventory_data_to_pandas(
+                batch_data, *FEATURE_LIST)
+            data_frame, _mapping = rad.preprocess(data_frame)
+        with METRICS['processing_time'].time():
+            isolation_forest = rad.IsolationForest(
+                data_frame,
+                num_trees,
+                sample_size,
```

Comment:
Do the time metrics here overwrite the reading taken in the previous run?
When I executed the pipeline the seconds jumped from 77.84 to 77.95 as shown below -

<img width="1420" alt="Screen Shot 2019-05-09 at 4 41 03 PM" src="https://user-images.githubusercontent.com/1538216/57493329-48d46880-7279-11e9-9273-f2d4c4c3b20a.png">

Not sure how to interpret the graph above